packages/angular/src/generators/ng-add/utilities/normalize-options.ts

```diff
@@ -35,6 +35,13 @@ export function normalizeOptions(
   if (!npmScope) {
     // use the name (scope if exists) in the root package.json
     const { name } = readJson(tree, 'package.json');
+
+    if (!name) {
+      throw new Error(
+        'Add the name-property to your package.json and try again.'
+      );
+    }
+
     npmScope = name.startsWith('@') ? name.split('/')[0].substring(1) : name;
```

Comment:
We don't need to throw. Not having a name shouldn't block the migration. It's not critical or required to have the `npmScope` value set. So let's change this block to only set the `npmScope` when the `name` is available.

```suggestion
    if (name) {
      npmScope = name.startsWith('@') ? name.split('/')[0].substring(1) : name;
    }
```